assets/loc/translations.csv

```diff
@@ -349,7 +350,7 @@ irys,IRyS,IRyS,IRyS,IRyS
 kazama_iroha,Kazama Iroha,카자마 이로하,風真いろは,Kazama Iroha
 kobo_kanaeru,Kobo Kanaeru,코보 카나에루,こぼ・かなえる,Kobo Kanaeru
 kureiji_ollie,Kureiji Ollie,Kureiji Ollie,クレイジー・オリー,Kureiji Ollie
-laplus_darkness,La+ Darknesss,La+ Darknesss,ラプラス・ダークネス,La+ Darknesss
+laplus_darkness,Laplus Darknesss,Laplus Darknesss,ラプラス・ダークネス,Laplus Darknesss
```

Comment:
was this + messing something up? I think it's supposed to be La+ ?

Reply:
Just for ease of searching. I found typing La+ too cumbersone.